### PR TITLE
Mensagia (Independent Publisher) v1.0

### PR DESCRIPTION
**Pull request by AfriSR**
---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

- [x] I attest that the connector works and I verified by deploying and testing all the operations. 
- [x] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [x] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [x] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [x] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [x] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [x] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [x] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md. 

Create contact Action
Create Contact Success
Send SMS Action
Send SMS Success
Send Transactional Email Action
Send Transactional Email success

### CODE REVIEW

File: independent-publisher-connectors/Mensagia/apiDefinition.swagger.json [hunk 1]
```diff
@@ -0,0 +1,2330 @@
+{
+    "swagger": "2.0",
+    "info": {
+        "title": "Mensagia",
+        "description": "Mensagia offer reliable SMS, Email and Voice for business automation communication.",
+        "version": "1.0",
+        "contact": {
+            "name": "Sinergyne Global Communications, S.L.",
+            "email": "support@mensagia.com",
```
Comment (sriyen-msft) [inline independent-publisher-connectors/Mensagia/apiDefinition.swagger.json:9]: @AfriSR We need to support contact for your connector, not for the service/API.

File: independent-publisher-connectors/Mensagia/apiDefinition.swagger.json [hunk 2]
```diff
@@ -0,0 +1,2330 @@
...
+    "responses": {},
+    "securityDefinitions": {
+        "Clave de API": {
+            "type": "apiKey",
+            "in": "query",
+            "name": "api_token"
+        }
+    },
+    "security": [{
+        "Clave de API": []
```
Comment (sriyen-msft) [inline independent-publisher-connectors/Mensagia/apiDefinition.swagger.json:2327]: @AfriSR We do not support this security type, you can remove the entire security section.

File: independent-publisher-connectors/Mensagia/apiProperties.json [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+                        "required": "true"
+                    }
+                }
+            }
+        },
+        "iconBrandColor": "#FDFDFD",
+        "capabilities": [
+            "actions"
+        ],
+        "publisher": "Sinergyne Global Communications, S.L.",
```
Comment (sriyen-msft) [inline independent-publisher-connectors/Mensagia/apiProperties.json:22]: @AfriSR Please provide your name as a publisher as you are publishing it as an independent publisher connector.

File: independent-publisher-connectors/Mensagia/Readme.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+## Publisher: Sinergyne Global Communications, S.L.
+
+## Prerequisites
+You will need the following to proceed:
+* A Mensagia "API Client" account.
+* Make sure you have enough balance in your Mensagia account.
+
+## Obtaining Credentials
+The API key can be obtained by entering the portal of your Mensagia account.
```
Comment (sriyen-msft) [inline independent-publisher-connectors/Mensagia/Readme.md:12]: @AfriSR You need to provide step-by-step details to get an API that can be understood by everyone.

File: independent-publisher-connectors/Mensagia/Readme.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
+# Mensagia
+Mensagia offer reliable SMS, Email and Voice for business automation communication.
+
+## Publisher: Sinergyne Global Communications, S.L.
```
Comment (sriyen-msft) [inline independent-publisher-connectors/Mensagia/Readme.md:4]: @AfriSR Please provide your name as a publisher as you are publishing it as an independent publisher connector.

File: independent-publisher-connectors/Mensagia/settings.json [hunk 6]
```diff
@@ -0,0 +1,9 @@
+{
```
Comment (sriyen-msft) [inline independent-publisher-connectors/Mensagia/settings.json:1]: @AfriSR The `settings.json` is not a required file to open source, please remove it from your PR.

File: independent-publisher-connectors/Mensagia/apiProperties.json [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+                    "tooltip": "Set your Api Key.",
+                    "constraints": {
+                        "tabIndex": 2,
+                        "clearText": false,
+                        "required": "true"
+                    }
+                }
+            }
+        },
+        "iconBrandColor": "#FDFDFD",
```
Comment (sriyen-msft) [inline independent-publisher-connectors/Mensagia/apiProperties.json:18]: @AfriSR You can find [a template](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/Independent%20Publisher/apiProperties.json) for `apiProperties.json` and default iconBrandColor for any IP connector is `"da3b01"`

File: independent-publisher-connectors/Mensagia/apiDefinition.swagger.json [hunk 8]
```diff
@@ -0,0 +1,2330 @@
...
+                    }
+                ]
+            }
+        }
+    },
+    "definitions": {},
+    "parameters": {},
+    "responses": {},
+    "securityDefinitions": {
+        "Clave de API": {
```
Comment (sriyen-msft) [inline independent-publisher-connectors/Mensagia/apiDefinition.swagger.json:2320]: @AfriSR Is it supposed to be `"api_key"`?

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

**Review by sriyen-msft — CHANGES_REQUESTED** [review]
@AfriSR And please sign CLA to get this PR reviewed and approved.

File: independent-publisher-connectors/Mensagia/apiDefinition.swagger.json [hunk 9]
```diff
@@ -0,0 +1,2330 @@
...
+        "description": "Mensagia offer reliable SMS, Email and Voice for business automation communication.",
+        "version": "1.0",
+        "contact": {
+            "name": "Sistemes Informàtics ICON, S.L.",
+            "email": "asanteliz@iconsl.com",
+            "url": "https://www.iconsl.com/"
+        }
+    },
+    "x-ms-connector-metadata": [{
+            "propertyName": "Website",
```
Comment (sriyen-msft) [inline independent-publisher-connectors/Mensagia/apiDefinition.swagger.json:14]: @AfriSR Indentation - The formatting of both the swagger and apiProperties.json files is not correct. The properties are not aligned properly and there is a longer tab index between the properties. Please refer to the certified connectors in the [open source repo](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/certified-connectors/Africa%27s%20Talking%20Airtime/apiDefinition.swagger.json) for reference.

File: independent-publisher-connectors/Mensagia/apiProperties.json [hunk 10]
```diff
@@ -0,0 +1,25 @@
...
+                    "tooltip": "Set your Api Key.",
+                    "constraints": {
+                        "tabIndex": 2,
+                        "clearText": false,
+                        "required": "true"
+                    }
+                }
+            }
+        },
+        "iconBrandColor": "#f8f8f8",
```
Comment (sriyen-msft) [inline independent-publisher-connectors/Mensagia/apiProperties.json:18]: @AfriSR You can not set a different iconBrandColor for an independent publisher connector. Please change it to `"da3b01"`.

A [template](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/Independent%20Publisher/apiProperties.json) for apiProperties.json

File: independent-publisher-connectors/Mensagia/apiProperties.json [hunk 11]
```diff
@@ -0,0 +1,25 @@
...
+        "capabilities": [
+            "actions"
+        ],
+        "publisher": "Sistemes Informàtics ICON, S.L.",
```
Comment (sriyen-msft) [inline independent-publisher-connectors/Mensagia/apiProperties.json:22]: @AfriSR The publisher value has unicodes which may not be supported in all the Power Automate pages, Can you please replace all the unicodes with respective characters in the English language.

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

**Comment by sriyen-msft** [comment]
[[certify-connector]]

**Comment by sriyen-msft** [comment]
@AfriSR Update - Your connector is now in production deployment and reached to [Test](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#region-deployment) region, and it would be completed and available in all the production regions by Thursday 18th October 2021. Please let us know if you have any questions. Thanks

**Comment by sriyen-msft** [comment]
@AfriSR Deployment freeze notice: Due to Thanksgiving week in the United States, we have temporarily halted the deployments of all the connectors till Tuesday, 30th November 2021. So we will not be able to deploy your connector to the next production region.

We will begin the deployments on Wednesday, 1st December 2021, and the production deployment of your connector in all regions will be expected to be completed by Thursday, 2nd December 2021, Thanks

**Comment by AfricaSR** [comment]
> @AfriSR Deployment freeze notice: Due to Thanksgiving week in the United States, we have temporarily halted the deployments of all the connectors till Tuesday, 30th November 2021. So we will not be able to deploy your connector to the next production region.
> 
> We will begin the deployments on Wednesday, 1st December 2021, and the production deployment of your connector in all regions will be expected to be completed by Thursday, 2nd December 2021, Thanks

Hi, is there anything new with the connector deployment? Thanks!

**Comment by sriyen-msft** [comment]
@AfricaSR Sorry for the delay in response, We had a long deployment freeze till the last week.

**Comment by sriyen-msft** [comment]
@AfriSR Congratulations! The deployment of your connector and documentation page is now complete, We have a short [survey](https://forms.office.com/Pages/ResponsePage.aspx?id=v4j5cvGGr0GRqy180BHbR0BHMrjhL0hDmckROosW6AFURERIVUUzMk5XRzJGV1dJM1Q5RVZJUjFXSC4u) on the connector certification process that we would like for you to complete. Your feedback is appreciated and will help us improve the program. Thanks

**Review by sriyen-msft — APPROVED** [review]

**Comment by AfricaSR** [comment]
Hello! thanks for the deployment confirmation! Where can we check the connector's publication page?

Have a nice day!

**Pull request merged**